nservicebus/azure/understanding-transactionality-in-azure.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Understanding Transactionality in Azure
+summary: Understanding transactions in Azure and NServiceBus.
+tags:
+- Azure
+- Cloud
+- Transactions
+- Idempotency
+- DTC
+redirects:
+ - nservicebus/transactions
+ - nservicebus/understanding-transactions-in-windows-azure
+reviewed: 2016-04-20
+---
+
+The Azure Platform and NServiceBus complement each other. Azure is a distributed, scalable and flexible platform. NServiceBus provides high level abstractions and features that make development in Azure easier.
```

Comment:
It's already existing content that I moved ( split into 2 files ), but feel free to review when time allows